ChatSample/ChatSample/Startup.cs

```diff
@@ -29,7 +30,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseSignalR(routes =>
             {
-                routes.MapHub<ChatHub>("chat");
+                routes.MapHub<ChatHub>("chat", options => { options.Transports = TransportType.All; });
```

Comment:
Why this change?

Reply:
I used this to test fallback and decided to leave to show that you can pass options to MapHub. I don't think we have this in any of the samples.

Reply:
We should remove it. It's ugly and not required. If we had a sample showing specifically how to only use support websockets, it would be more useful.

Reply:
OK

Reply:
Do you any other transport fallback protocol sample?
I'm creating my first SignalR functionality on a .netcore + ng4 app and I would really appreciate any sample that can help me leverage the problem of determining an alternative transport protocol.
Regards

Reply:
@asfaya - why would you need a fallback for transport protocol? Currently, the server supports both MsgPack and JSon so, whichever you pick you will be fine.

Reply:
While running the app from kestrel everything works fine.
If, from VS, I run the app on IIS Express, occurs and error because the client tries to use websockets and the server is not responding. And there is no "automatic fallback" to long pooling (for instance)
So, I was thinking in trying to catch the error when not being able to connect with websockets and try again using long pooling.

Reply:
Are you on windows 7?

Reply:
Hi David, sorry for the delay. Yes, I'm using Win7 x64
VS 2017 + IIS Express 

Thanks!

Andrés

Reply:
If you know your server does not support webSockets (i.e. you are running behind IIS/IISExpress on Win7) you can disable the WebSockets transport on the server side. This is basically https://github.com/aspnet/SignalR/issues/1033

Reply:
I'll check that out. Thank you!